### PR TITLE
New trivia solver

### PR DESCRIPTION
**Pull request by ElOxo**
Add new TriviaRoom Solver to Project. Also i added a gitignore File ;)

### CODE REVIEW

File: features/triviaSolver.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class TriviaSolver {
+    constructor() {
+        this.name = "Trivia Solver";
+        this.triggers = {}
+        this.run();
+    }
+
+run() {
```
Comment (Antonio32A) [inline features/triviaSolver.js]: missing indent

File: features/triviaSolver.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    let trivia = JSON.parse(FileLib.read("DungeonUtilities", "trivia.json"));
+
```
Comment (Antonio32A) [inline features/triviaSolver.js]: unnecessary newline

File: loader.js [hunk 3]
```diff
@@ -7,7 +7,8 @@ const features = [
     "partyKicker",
     "riddleSolver",
     "betterGlowingEffect",
-    "lootTracker"
+    "lootTracker",
+	"triviaSolver"
```
Comment (Antonio32A) [inline loader.js]: missing indent

**Review by Antonio32A — CHANGES_REQUESTED** [review]

**Comment by Antonio32A** [comment]
oh also credit SirDesco in the settings file too

**Comment by ElOxo** [comment]
Making a New 1 because of too many errors

**Pull request closed without merging**